src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingAdjustmentsMigrationServiceTest.kt

```diff
@@ -381,24 +381,25 @@ internal class SentencingAdjustmentsMigrationServiceTest {
       }
 
       @Test
-      internal fun `will check again in 10 second and reset even when previously started finishing up phase`(): Unit = runBlocking {
-        service.migrateStatusCheck(
-          MigrationContext(
-            type = SENTENCING_ADJUSTMENTS,
-            migrationId = "2020-05-23T11:30:00",
-            estimatedCount = 100_200,
-            body = MigrationStatusCheck(checkCount = 4)
+      internal fun `will check again in 10 second and reset even when previously started finishing up phase`(): Unit =
+        runBlocking {
```

Comment:
how come you need to wrap all these in `runBlocking` what had changed?

Reply:
looks like a mistake - will check